content/post/085-announcing-rust-ipfs.md

```diff
@@ -0,0 +1,62 @@
+---
+date: 2020-03-16
+url: 2020-03-16-announcing-rust-ipfs
+title: Announcing Rust IPFS, and a call for contributors
+author: Mark Robert Henderson and Molly Mackinlay
+---
+
+![Rust IPFS Logo](https://github.com/ipfs-rust/logo/raw/master/rust-ipfs-logo-256w.png)
+
+Calling all rustaceans, rustafarians, ferrosities, and rustlers - we’ve got an exciting update!
+Active full time work on a rust-IPFS implementation has commenced, building on the great work by
+Parity in `rust-libp2p`. [Equilbrium](https://equilibrium.co) is spearheading the new community and
+implementation with support from Protocol Labs, and is looking for additional rust devs itching
+to help build a new language implementation of the InterPlanetary File System combining the
+performance and resource utilization benefits of Rust with a keen eye on conformance to the IPFS spec.
+
+## Why Rust IPFS? Why Now?
+
+Rust, the programming language, has enjoyed a recent spike in popularity. This is due both to its
+inclusive community, and also being a safe systems language with performance comparable to C and
+C++. An IPFS implementation written in Rust only makes sense. Additionally, the community has been
+[asking for a while now](https://github.com/ipfs/notes/issues/363), for a number of use cases:
+
+1. Usage in resource-constrained systems such as Industrial Internet-of-Things (IIoT) controllers
+2. Portability of certain subsets of the IPFS stack into WebAssembly
+3. Usage of IPFS functionality and APIs via native Rust function calls via inclusion as a Rust crate.
+
+## Leveraging Community Efforts
+
+The IPFS community has been and remains an impressive, talented group of people who have
+astounded and impressed us with their work in the Rust space already. We have the aforementioned
+[rust-libp2p](https://github.com/libp2p/rust-libp2p), which is already used by the likes of
+Polkadot, Substrate, and Lighthouse. We have ferrismtg's
+[rust-ipfs-api](https://github.com/ferristseng/rust-ipfs-api) which is aleady doing wonders in
+providing a bridge from the Rust world. Finally, we have David Craven's
+[rust-ipfs](https://github.com/ipfs-rust/rust-ipfs) foundations, which is the main repo this grant will build upon.
+
+This devgrant endeavors to take this impressive amount of community work and carry the torch across
```

Comment:
```suggestion
Instead of forking and/or starting fresh, this project aims to support and build on the impressive amount of progress that's already been made, helping carry the torch across the finish line while _incorporating_ and _including_ the community of passionate Rust contributors.
```